weekly-openmeteo/conftest.py

```diff
@@ -38,5 +39,72 @@ def invalid_response_type():
 
 @pytest.fixture
 def valid_coords():
-    """Fixture for invalid API response data."""
+    """Fixture for valid coords."""
```

Comment:
Fixtures are good, could keep in format with the others and use `'''`s rather than `"""`s here though